apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Snippets/GoogleStackdriverAppenderSnippets.cs

```diff
@@ -68,16 +73,37 @@ public void Overview()
                 ILog log = LogManager.GetLogger(typeof(Program));
                 // Log some information. This log entry will be sent to Google Stackdriver Logging.
                 log.Info("An exciting log entry!");
+
+                // Flush buffered log entries before program exit.
+                // This is required because log entries are buffered locally before being sent to StackDriver.
+                // LogManager.Flush() only works in the full .NET framework (not in .NET Core):
+                bool flushCompleted = LogManager.Flush(10_000);
+                // On .NET Core, the specific repository needs to be flushed:
+                bool repositoryFlushCompleted = ((IFlushable)LogManager.GetRepository(GetType().GetTypeInfo().Assembly)).Flush(10_000);
                 // End sample
+
+                Assert.True(repositoryFlushCompleted);
+
+                var logClient = LoggingServiceV2Client.Create();
+                var logName = new LogName(projectId, logId);
+                // Wait up to 30 seconds for the log entry to appear in StackDriver.
+                for (int i = 0; i < 30; i += 1)
```

Comment:
Any reason for `i += 1` rather than `i++`?

Reply:
Not really. Changed to `i++`

Reply:
Great, thanks :)